src/common/components/page-links/index.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component, PropTypes } from 'react';
+
+import styles from './styles.css';
+
+const LINK_LABEL_OPTIONS = ['first', 'prev', 'next', 'last'];
+
+export default class PageLinks extends Component {
+
+  render() {
+    const renderLink = this.renderLink.bind(this);
+
+    let links = LINK_LABEL_OPTIONS.map((item) => {
+      return renderLink(
+        this.props[item],
+        item
+      );
+    });
+
+    return (
+      <ul className={ styles.container }>
+        { links }
+      </ul>
+    );
+  }
+
+  renderLink(item, label) {
+    let url;
+
+    if (typeof item == 'string') {
```

Comment:
The line I tend to draw is that if the comment is purely about the change then it can go in the PR, but if it's about the new state of the code then it's better to put it in the code because it might equally be useful for people trying to understand the code in future.